app/src/components/search/RadiusSelect.vue

```diff
@@ -2,30 +2,28 @@
   <v-select
     class="radius_select"
     label="Umkreis"
-    item-value="value"
     :items="radii"
     :dark="dark"
     :disabled="disabled"
     :attach="attachTo"
-    v-model="radius"
-    @change="onInputChange"
+    :value="radius"
+    @change="onChange"
     @keydown.enter="onEnter"
   />
 </template>
 
 <script lang="ts">
 import Vue from "vue";
 import Radius from "@/models/radius";
-import radii from "@/resources/radii";
+import radii, { radii as radiiArray } from "@/resources/radii";
```

Comment:
Why do we import radii and radii as radiArray?
I think it would be less confusing if use the default export and rename the radii-variable of this component.